Update RedditBridge.php

Tone866 opened the pull request:
I read my rss mostly on mobile and old.reddit.com isn't very good to read on mobile devices.
So I added an option to change the article link to the "new" reddit.

Tone866 committed:
Update RedditBridge.php

added an option to change the article link to the "new" reddit (for mobile users)

Tone866 committed:
Update RedditBridge.php

Tone866 committed:
Update RedditBridge.php

dvikan commented:
good idea.

can you change this into a list? because there are multiple frontends e.g. also https://libreddit.kavin.rocks/r/python

Tone866 committed:
Update RedditBridge.php

changed the option to choose the new reddit interface to a more general list for frontends

Tone866 committed:
Update RedditBridge.php

Tone866 committed:
Update RedditBridge.php

Tone866 commented:
> good idea.
> 
> can you change this into a list? because there are multiple frontends e.g. also https://libreddit.kavin.rocks/r/python

Ok, I changed it to a more generic list for all frontends

dvikan commented:
this pr breaks existing feed urls because they dont have the frontend parameter.

Tone866 committed:
Update RedditBridge.php

prevent breaking existing feeds without $frontend

Tone866 commented:
existing feeds should now work as well

dvikan committed:
lint
